pkg/yurthub/certificate/token/token.go

```diff
@@ -274,6 +328,9 @@ func (ycm *yurtHubCertManager) getPkiDir() string {
 
 // getBootstrapConfFile returns the path of yurthub bootstrap conf file
 func (ycm *yurtHubCertManager) getBootstrapConfFile() string {
+	if len(ycm.bootstrapFile) != 0 {
+		return ycm.bootstrapFile
+	}
```

Comment:
`generateCertClientFn` will invoke `getBootstrapConfFile()` function, dose it work well if yurthub restart and bootstrap file doesn't exist?

Reply:
yes, we need to use bootstrap file to apply certificates when certificates expired, so getBootstrapConfFile() is used in generateCertClientFn, and yurthub will always wait for bootstrap file until it is prepared by `yurtadm renew certificate` command.